packages/sit-onyx/src/components/OnyxBadge/types.ts

```diff
@@ -0,0 +1,18 @@
+import type { DensityProp } from "../../composables/density";
+
+export type OnyxBadgeProps = DensityProp & {
+  /**
+   * The color of the badge.
+   */
+  variation?: BadgeVariation;
+};
+
+export const BADGE_VARIATIONS = [
```

Comment:
We already have a type `OnyxColor` for this :) It also includes neutral which we should add styles for to have the full color palette available